doc/design/staging-1.md

```diff
@@ -48,4 +48,4 @@ A developer has a feature they want to add that only impacts the backend.
 * Once everything is tested, tags a new release of `dandi-api`, which is then automatically deployed to production.
 
 ## TODO
-Figure out a less manual release procedure for `dandiarchive`.
+Figure out a less manual release procedure for `dandi-api`.
```

Comment:
We don't want a "less manual" procedure for `dandi-api`, do we?

Reply:
I was correcting what I thought was a typo in the original design doc, since it is `dandi-api` that has a manual release procedure and not `dandiarchive`. I'm not sure if we still want a less manual procedure, @dchiquito is this statement outdated? FWIW I have no issues with the current release procedure.

Reply:
By manual do you mean "mint a release"? That could be automated via PRs labeling if desired, like we do in dandi-cli and dandischema with auto

Reply:
But what I wonder - is it possible to unify and avoid staging branch for webui? IMHO such unique discrepancy (all other components by now rely on master + releases) is just prone to eventually put in production some intended for staging change

Reply:
We are going to leave our staging/production setup as-is for now, until some problem with it arises in practice. (Though I like consistency in general, that's not reason enough for us to change how the frontend is deployed, due to some difficulties around how Netlify works.)

Reply:
If everyone is happy with how the release procedure currently works (I am), then I don't think there's anything left TODO and this comment can be deleted.